### PR TITLE
Api router and module was added

### PR DESCRIPTION
**Pull request by ovekyc**
We added 'body-parser' module for json 
and 'levelup', 'leveldown' for leveldb

New files are about api routes and leveldb. 

@fguy @shberry16 @ochanje210 @yuda1124

### CODE REVIEW

File: src/server/items/items-routes.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import express from 'express';
+
+const router = express.Router();
+
+router.get('/', (req, res, next) => {
```
Comment (fguy) [inline src/server/items/items-routes.js:5]: I believe you will implement these endpoints soon. Don't forget to add tests.

File: src/server/routes/api-routes.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import express from 'express';
+import items from '../items/items-routes';
+const router = express.Router();
```
Comment (fguy) [inline src/server/routes/api-routes.js:4]: A new line needed between import and const.

File: src/server/items/items-routes.js (hunk 1 above)
Comment (ovekyc) [reply src/server/items/items-routes.js:5]: We will! First we specify the REST with Swagger and then we will fill those functions

**Comment by fguy** [comment]
LGTM

File: src/server/items/items-routes.js (hunk 1 above)
Comment (fguy) [reply src/server/items/items-routes.js:5]: You can add empty success tests now though.

File: src/server/routes/api-routes.js (hunk 2 above)
Comment (ovekyc) [reply src/server/routes/api-routes.js:4]: Occur error with newline! 
but I dont know why...

**Pull request merged**